src/api_server.cc

```diff
@@ -47,7 +47,12 @@ void MetadataApiServer::Handler::operator()(const HttpServer::request& request,
       if (config_.VerboseLogging()) {
         LOG(INFO) << "Found resource for " << id << ": " << resource;
       }
+      std::map<std::string, std::string> headers = {
+        {"Content-Type", "application/json"},
+      };
+
       conn->set_status(HttpServer::connection::ok);
+      conn->set_headers(headers);
```

Comment:
Would the following work:
```cpp
      conn->set_headers({
        {"Content-Type", "application/json"},
      });
```
? Also below.

Reply:
I've inlined it, using the syntax the way you had it resulted in the error below. Is this a happy medium?

```
api_server.cc:53:8: error: no matching function for call to 'boost::network::http::async_connection<boost::network::http::tags::http_server, google::MetadataApiServer::Handler>::set_headers(<brace-enclosed initializer list>)'
```

Reply:
Yeah, wrapping it into an `std::map` temporary is exactly the right fix for that. Sorry I didn't check this before suggesting.